projects/f3d.suffix.cmake

```diff
@@ -10,10 +10,18 @@ else ()
     "${CMAKE_SYSTEM_NAME}")
 endif ()
 
-if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" MATCHES "^(x86_64||AMD64)$")
-  list(APPEND package_suffix_items "x86_64")
-else()
-  list(APPEND package_suffix_items "unknown")
+list(LENGTH "${CMAKE_OSX_ARCHITECTURES}" osx_arch_count)
+
+if (NOT APPLE OR "${osx_arch_count}" EQUAL 0)
+  if ("${CMAKE_HOST_SYSTEM_PROCESSOR}" MATCHES "^(x86_64||AMD64)$")
+    list(APPEND package_suffix_items "x86_64")
+  else()
+    list(APPEND package_suffix_items "unknown")
+  endif()
+elseif ("${osx_arch_count}" EQUAL 1)
+  list(APPEND package_suffix_items "${osx_arch_count}")
+else ()
+  list(APPEND package_suffix_items "universal")
```

Comment:
is it actually universal ? I think its specific to arm, is it not ?

Reply:
You can set `CMAKE_OSX_ARCHITECTURES` to `x86_64;arm64` is which case macos generates binaries compatible with both CPUs. They call that [Universal](https://developer.apple.com/documentation/apple-silicon/building-a-universal-macos-binary) but it's not convenient since the binary size will basically double.

Reply:
But do we support building a package like this ? I dont think so.

Reply:
I think we support it, I'll try

Reply:
Looks like more work is needed to support `universal`. I'll just remove what I started and leave it as future work